internal/api/annotations/annotations.go

```diff
@@ -1,8 +1,27 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package annotations
 
 const (
 	SELinuxPolicyConfigAnnotation = "config.node.openshift-kni.io/selinux-policy"
 	SELinuxPolicyCustom           = "custom"
+	// MultiplePoolsPerTreeAnnotation an annotation used to re-enable the support of multiple node pools per tree; starting 4.18 it is disabled by default
+	MultiplePoolsPerTreeAnnotation = "experimental.multiple-pools-per-tree"
+	MultiplePoolsPerTreeEnabled    = "enabled"
```

Comment:
suggestions are welcome for a better name of the annotation. 
chose this one after following the syntax here:https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set
this annotation should be private and used by the user and the not system componenet, so no DNS subdomain is needed.

Reply:
I'd go with `config.node.openshift-kni.io/multiple-pools-per-tree`
the allowed values could be `disabled` and `enabled`